OIPA/iati/parser/validators.py

```diff
@@ -1530,6 +1538,22 @@ def activity_transaction(
                     apiField="transaction_type.code",
                     ))
 
+        if not disbursement_channel_code:
+            errors.append(
+                RequiredFieldError(
```

Comment:
@VincentVW . I have changed it to not required Field. Thanks for your feedback. I'll merge!